### PR TITLE
ref: Fix ruff rules in preview mode

### PR DESCRIPTION
**Pull request by cbornet**
Fix ruff rules in preview mode.
Note: we don't use preview mode in CI (for good reason)

### CODE REVIEW

File: src/backend/base/langflow/custom/custom_component/component.py [hunk 1]
```diff
@@ -588,9 +588,9 @@ def _update_template(self, frontend_node: dict):
         return frontend_node
 
     def to_frontend_node(self):
-        #! This part here is clunky but we need it like this for
-        #! backwards compatibility. We can change how prompt component
-        #! works and then update this later
+        # ! This part here is clunky but we need it like this for
```
Comment (cbornet) [inline src/backend/base/langflow/custom/custom_component/component.py:591]: Is the syntax `#!` used by something ? (conflicts with shebang btw)

File: src/backend/base/langflow/services/settings/base.py [hunk 2]
```diff
@@ -266,14 +266,14 @@ def set_database_url(cls, value, info):
                     if Path(new_path).exists():
                         logger.debug(f"Database already exists at {new_path}, using it")
                         final_path = new_path
-                    elif Path("./{db_file_name}").exists():
+                    elif Path(f"./{db_file_name}").exists():
```
Comment (cbornet) [inline src/backend/base/langflow/services/settings/base.py:269]: FIxing a bug here ?

File: src/backend/base/langflow/custom/custom_component/component.py (hunk 1 above)
Comment (ogabrielluiz) [reply src/backend/base/langflow/custom/custom_component/component.py:591]: Not really. I use https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments and that makes it pop a bit more but not necessary. It is just to make sure people pay attention to certain comments more than others.

File: src/backend/base/langflow/services/settings/base.py (hunk 2 above)
Comment (ogabrielluiz) [reply src/backend/base/langflow/services/settings/base.py:269]: yes! Thank you

File: src/backend/base/langflow/custom/custom_component/component.py (hunk 1 above)
Comment (cbornet) [reply src/backend/base/langflow/custom/custom_component/component.py:591]: Does it still highlight if you have a space before the `!` ?

Comment (cbornet) [reply src/backend/base/langflow/custom/custom_component/component.py:591]: I think it should work and is recommended to have the space: https://github.com/aaron-bond/better-comments/issues/8#issuecomment-317007002

**Comment by cbornet** [comment]
Added support for `EXE` rule as it passes once `# !` is used instead of `#!`

File: src/backend/base/langflow/custom/custom_component/component.py (hunk 1 above)
Comment (ogabrielluiz) [reply src/backend/base/langflow/custom/custom_component/component.py:593]: Go for it. Thanks for the heads up!

**Review by ogabrielluiz — APPROVED** [review]
LGTM

**Pull request merged**